common_dwp_api/src/test/java/uk/gov/dhsc/htbhf/dwp/model/QualifyingReasonTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dhsc.htbhf.dwp.model;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class QualifyingReasonTest {
+
+    @ParameterizedTest
+    @CsvSource({
```

Comment:
Child tax credit is missing.

Reply:
added now in both QualifyingBenefitsTest and QualifyingReasonTest